basic.obo

```diff
@@ -219494,39 +220777,189 @@ created_by: http://orcid.org/0000-0001-9897-3238
 
 [Term]
 id: UBERON:8400001
-name: hepatic acinus periportal zone
-def: "The region of the hepatic acinus closest to the portal triad. Blood in this zone has the highest oxygen concentration of the three metabolic zones in the hepatic acinus." [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
-synonym: "hepatic acinus zone 1" EXACT [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
-is_a: UBERON:0000063 ! organ subunit
-is_a: UBERON:0004119 ! endoderm-derived structure
+name: hepatic acinus zone 1
+def: "The region of the hepatic acinus that is closest to the arterial blood supply and has the highest oxygen concentration. Hepatocytes in this region are primarily involved in oxidative energy metabolism." [http://www.ncbi.nlm.nih.gov/pubmed/29763190]
+synonym: "liver lobule periportal zone" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+is_a: UBERON:8400005 ! metabolic zone of liver
```

Comment:
Original name not retained as synonym.  @mkeays - must have missed this in PR review.

Reply:
@dosumis e.g. "hepatic acinus periportal zone" ? Yes, I think it makes sense to include after all as it does get used, although it seems technically it's sort of a confusion between structural and functional concepts. Can add back as a synonym (also the below ones). Should I just do this on `master` or create a new branch?

Reply:
Have fixed on branch `hepatic_acinus_zone_synonyms` (https://github.com/obophenotype/uberon/pull/1791)